go/pkg/sysdb/coordinator/table_catalog.go

```diff
@@ -216,6 +217,7 @@ func (tc *Catalog) GetAllTenants(ctx context.Context, ts types.Timestamp) ([]*mo
 	return result, nil
 }
 
+// If collection is soft-deleted, then hard delete it, before creating a collection with the same name.
```

Comment:
hmmm should we do that / is that desirable? i guess thats the only sensible behavior

Reply:
I agree that we'd somehow keep the row around.  I'm just reading this code now, but is there some way to leave the uuid-identified row around while renaming the user-visible string?

What is it that we want from soft-deletes?  Is it compromised by this choice?